ipa-core/src/protocol/ipa_prf/shuffle/base.rs

```diff
@@ -92,7 +92,7 @@ where
     let mut x_2 = x_1.clone();
     add_single_shares_in_place(&mut x_2, z_31);
     x_2.shuffle(&mut rng_perm_l);
-    send_to_peer(&x_2, ctx, &OPRFShuffleStep::TransferX2, Direction::Right).await?;
+    send_to_peer(&x_2, ctx, &OPRFShuffleStep::TransferXY, Direction::Right).await?;
```

Comment:
To allow interceptor reuse between the sharded and non-sharded tests, steps are renamed to make `TransferXY` and `TransferC` have the same names for `OPRFShuffleStep` and `ShardedShuffleStep`.